FileAPI/blob/Blob-slice-overflow.html

```diff
@@ -16,26 +16,26 @@
 
 test(function() {
   var blob = new Blob([text]);
-  var sliceBlob = blob.slice(-1, 2000);
-  assert_equals(sliceBlob.size, 2000-(2000-1), "Bolb slice size");
+  var sliceBlob = blob.slice(-1, blob.size);
+  assert_equals(sliceBlob.size, blob.size-(blob.size-1), "Blob slice size");
 }, "slice start is negative, relativeStart will be max((size + start), 0)");
 
 test(function() {
   var blob = new Blob([text]);
-  var sliceBlob = blob.slice(2001, 2000);
-  assert_equals(sliceBlob.size, 0, "Bolb slice size");
+  var sliceBlob = blob.slice(blob.size+1, blob.size);
+  assert_equals(sliceBlob.size, 0, "Blob slice size");
 }, "slice start is greater than blob size, relativeStart will be min(start, size)");
 
 test(function() {
   var blob = new Blob([text]);
-  var sliceBlob = blob.slice(1998, -1);
-  assert_equals(sliceBlob.size, (2000-1)-1998, "Bolb slice size");
+  var sliceBlob = blob.slice(blob.size-2, -1);
+  assert_equals(sliceBlob.size, (blob.size-1)-1998, "Blob slice size");
```

Comment:
Don't you want to replace the `1998` here too? Should be `blob.size - 2`.... or you could again replace the whole expected value with `1`